### PR TITLE
Lynx: Swapping blocks in patterns

### PR DESCRIPTION
**Pull request by alaczek**
Replacing profile image, heading, and paragraph blocks with site logo, title, and tagline in patterns.

See p1661499387489309-slack-C02M88KJ684 for background to this decision.

### CODE REVIEW

**Review by arthur791004 — APPROVED** [review]

File: lynx/patterns/link-in-bio-cloud.php [hunk 1]
```diff
@@ -1,18 +1,15 @@
...
  */
-
 ?>
 
 <!-- wp:cover {"overlayColor":"white","minHeight":100,"minHeightUnit":"vh","contentPosition":"center center","isDark":false,"align":"full"} -->
 <div class="wp-block-cover alignfull is-light" style="min-height:100vh"><span aria-hidden="true" class="wp-block-cover__background has-white-background-color has-background-dim-100 has-background-dim"></span><div class="wp-block-cover__inner-container"><!-- wp:group {"style":{"spacing":{"blockGap":"0px"}},"layout":{"inherit":false,"contentSize":"640px"}} -->
-<div class="wp-block-group"><!-- wp:paragraph {"align":"left","placeholder":"Write title…","style":{"color":{"text":"#00000099"}},"fontSize":"small"} -->
-<p class="has-text-align-left has-text-color has-small-font-size" style="color:#00000099"><?php echo  esc_html__( 'I’m Matt, an artist living in New York City.', 'lynx' ); ?></p>
-<!-- /wp:paragraph -->
+<div class="wp-block-group"><!-- wp:site-tagline {"style":{"color":{"text":"#00000099"}},"fontSize":"1rem"} /-->
```
Comment (mikachan) [inline lynx/patterns/link-in-bio-cloud.php:12]: Is this hex code correct: `#00000099`?

File: lynx/patterns/link-in-bio-columns-dark.php [hunk 2]
```diff
@@ -10,9 +10,7 @@
...
 <div class="wp-block-cover alignfull" style="padding-top:10vh;padding-right:5vw;padding-bottom:10vh;padding-left:5vw;min-height:100vh"><span aria-hidden="true" class="wp-block-cover__background has-black-background-color has-background-dim-100 has-background-dim"></span><div class="wp-block-cover__inner-container"><!-- wp:columns {"style":{"spacing":{"blockGap":"40%"}}} -->
 <div class="wp-block-columns"><!-- wp:column {"width":"360px","layout":{"inherit":true}} -->
-<div class="wp-block-column" style="flex-basis:360px"><!-- wp:paragraph {"style":{"color":{"text":"#ffffff99"}},"fontSize":"small"} -->
-<p class="has-text-color has-small-font-size" style="color:#ffffff99"><?php echo  esc_html__( 'Adam Jones is an artist living in New York City who has a passion for generative art in the 3D space. Currently accepting new clients.', 'lynx' ); ?></p>
-<!-- /wp:paragraph -->
+<div class="wp-block-column" style="flex-basis:360px"><!-- wp:site-tagline {"style":{"color":{"text":"#ffffff99"}},"fontSize":"small"} /-->
```
Comment (mikachan) [inline lynx/patterns/link-in-bio-columns-dark.php:13]: Is this hex code correct: `#ffffff99`?

File: lynx/patterns/link-in-bio-dark-background.php [hunk 3]
```diff
@@ -10,23 +10,17 @@
...
-<div class="wp-block-group"><!-- wp:image {"align":"left","width":100,"height":100,"sizeSlug":"large","linkDestination":"none"} -->
-<figure class="wp-block-image alignleft size-large is-resized"><img src="<?php echo esc_url( get_template_directory_uri() ) . '/assets/images/avatar_placeholder.png'; ?>" alt="" width="100" height="100"/></figure>
-<!-- /wp:image -->
+<div class="wp-block-group"><!-- wp:site-logo {"className":"is-style-rounded"} /-->
 
 <!-- wp:group {"style":{"spacing":{"blockGap":"0px"}},"layout":{"type":"flex","orientation":"vertical"}} -->
-<div class="wp-block-group"><!-- wp:paragraph {"style":{"typography":{"textTransform":"uppercase","letterSpacing":"2px","fontStyle":"normal","fontWeight":"500"},"color":{"text":"#ffffffb3"}}} -->
-<p class="has-text-color" style="color:#ffffffb3;font-style:normal;font-weight:500;letter-spacing:2px;text-transform:uppercase"><?php echo  esc_html__( 'Paul Nyberg', 'lynx' ); ?></p>
-<!-- /wp:paragraph -->
+<div class="wp-block-group"><!-- wp:site-title {"isLink":false,"style":{"color":{"text":"#ffffffb3"},"typography":{"textTransform":"uppercase","fontStyle":"normal","fontWeight":"500","letterSpacing":"2px"}}} /-->
```
Comment (mikachan) [inline lynx/patterns/link-in-bio-dark-background.php:16]: Same with this one: `#ffffffb3`?

File: lynx/patterns/link-in-bio-dark-background.php [hunk 4]
```diff
@@ -10,23 +10,17 @@
...
 
 <!-- wp:heading {"style":{"typography":{"textTransform":"uppercase","fontSize":"40px"},"color":{"text":"#dbff00"}},"className":"has-custom-font","fontFamily":"space-mono"} -->
 <h2 class="has-text-color has-space-mono-font-family has-custom-font" style="color:#dbff00;font-size:40px;text-transform:uppercase;"><?php echo  esc_html__( 'Musician &amp; Songwriter', 'lynx' ); ?></h2>
 <!-- /wp:heading --></div>
 <!-- /wp:group -->
 
-<!-- wp:paragraph {"style":{"color":{"text":"#ffffffb3"}},"fontSize":"small"} -->
-<p class="has-text-color has-small-font-size" style="color:#ffffffb3"><?php echo  esc_html__( 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.', 'lynx' ); ?></p>
-<!-- /wp:paragraph --></div>
+<!-- wp:site-tagline {"style":{"color":{"text":"#ffffffb3"}},"fontSize":"small"} /--></div>
```
Comment (mikachan) [inline lynx/patterns/link-in-bio-dark-background.php:23]: And this one: `#ffffffb3`

File: lynx/patterns/link-in-bio-large-heading.php [hunk 5]
```diff
@@ -9,15 +9,11 @@
...
-<p class="has-text-align-left has-text-color" style="color:#ed6e69;font-size:128px;font-style:normal;font-weight:900"><?php echo  esc_html__( 'Ella D.', 'lynx' ); ?></p>
-<!-- /wp:paragraph -->
+<div class="wp-block-group"><!-- wp:site-title {"isLink":false,"style":{"color":{"text":"#ed6e69"},"typography":{"fontSize":"128px","fontStyle":"normal","fontWeight":"900","lineHeight":"1"}}} /-->
 
 <!-- wp:columns -->
 <div class="wp-block-columns"><!-- wp:column {"width":"50%","style":{"spacing":{"blockGap":"10px"}}} -->
-<div class="wp-block-column" style="flex-basis:50%"><!-- wp:paragraph {"style":{"color":{"text":"#ffffffbf"}},"fontSize":"small"} -->
-<p class="has-text-color has-small-font-size" style="color:#ffffffbf"><?php echo  esc_html__( 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco.', 'lynx' ); ?></p>
-<!-- /wp:paragraph --></div>
+<div class="wp-block-column" style="flex-basis:50%"><!-- wp:site-tagline {"style":{"color":{"text":"#ffffffbf"}},"fontSize":"small"} /--></div>
```
Comment (mikachan) [inline lynx/patterns/link-in-bio-large-heading.php:16]: And this one: `#ffffffbf`

**Review by mikachan — COMMENTED** [review]
The changes to the patterns are looking good to me. 

I've left a few inline comments as I noticed that some color codes have more than 6 characters, but they're not part of the changes in this PR, so they could be fixed separately 👍

File: lynx/patterns/link-in-bio-cloud.php (hunk 1 above)
Comment (alaczek) [reply lynx/patterns/link-in-bio-cloud.php:12]: This color code was generated when I added opacity. This is what the editor generated, and seems to be working fine in the patterns.

Comment (mikachan) [reply lynx/patterns/link-in-bio-cloud.php:12]: Ahh OK, this must be an upstream bug then (hex codes should only be 3 or 6 characters long). I'll search for / open a GB issue.

It's probably working as I think the browser will default to black/#000000 (the first 6 characters).

Comment (alaczek) [reply lynx/patterns/link-in-bio-cloud.php:12]: > It's probably working as I think the browser will default to black/#000000 (the first 6 characters).

When I insert this pattern, the editor picks up the opacity setting correctly - doesn't switch to #000000.

Comment (alaczek) [reply lynx/patterns/link-in-bio-cloud.php:12]: I guess 8 digit HEX codes are a thing now: https://caniuse.com/?search=hex

Comment (mikachan) [reply lynx/patterns/link-in-bio-cloud.php:12]: Ooh I see, it's a way to set the alpha! I usually see it in this format: `rgba(0, 255, 0, 0.5)`. I'm less confused now, thanks! 😁

**Pull request merged**